packages/search-ui-analytics-plugin/src/index.ts

```diff
@@ -1,16 +1,118 @@
-import type { Event } from "@elastic/search-ui";
+import {
```

Comment:
could you open a PR to export these types in the tracker and import them here instead?

Reply:
yep!

Reply:
We need firstly to publish this package with 2.x branch and with exported types